### PR TITLE
Improve display of merged branches

### PR DESCRIPTION
**Pull request by mstv**
Fixes https://github.com/gitextensions/gitextensions/pull/8139#issuecomment-633742648, https://github.com/gitextensions/gitextensions/pull/8139#issuecomment-634106191

## Proposed changes

- Remove DPI info from Branch*Merged.png
- Handle artificial commits
- Suppress merged state for branches at the selected commit
- Wrap into Begin/EndUpdate

### Before

grafik

### After (ignore the tooltips, please)

grafik

grafik

## Test methodology 

- manual

## Test environment(s) 

- Git Extensions 3.3.0
- Build b613ba26577cd6262e108ae991a6ec2f9ae24fab
- Git 2.26.0.windows.1
- Microsoft Windows NT 6.2.9200.0
- .NET Framework 4.8.4180.0
- DPI 96dpi (no scaling)

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by gerhardol** [comment]
Note: Includes #8144 
Seem OK

File: GitUI/Strings.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@ internal sealed class Strings : Translate
         private readonly TranslationString _okText = new TranslationString("OK");
         private readonly TranslationString _cancelText = new TranslationString("Cancel");
 
+        private readonly TranslationString _containedInCurrentCommitText = new TranslationString("{0} is contained in the currently selected commit");
```
Comment (pmiossec) [inline GitUI/Strings.cs:16]: I don't know if it's not better to use `"'{0}' is contained in the currently selected commit"` (single quoted name of branch)?

**Comment by RussKie** [comment]
If we take any more strings we'll delay the release further. I have already asked the translators to do another pass.
I don't want to add any more strings in v3.4

**Comment by mstv** [comment]
> I don't want to add any more strings in v3.4

I move the tooltip to a separate PR.
Squashed and rebased on v3.4 without foreign commits.

**Pull request merged**